packaging/tools/update_rpcs_max_versions.py

```diff
@@ -0,0 +1,96 @@
+#!/usr/bin/env python
+import json
+import sys
+import re
+
+# This script updates the Apache Kafka RPCs max versions.
+# It reads the input from stdin, which should be a table
+# looking like the first three columns of the table in `INTRODUCTION.md`.
+# Should look like this (without the first space after the comment):
+# | 0       | Produce                       | 12         |
```

Comment:
I think it might be better to ask to copy paste all the columns, and the script handles it by modifying only the first three. 
This would reduce manual work first in removing the last column and then again adding it back when pasting in INTRODUCTION.md

Reply:
yeah, nice addition. Done